src/darwin/Framework/CHIP/templates/MatterClusterConstants.zapt

```diff
@@ -0,0 +1,76 @@
+{{> header}}
+
+#ifndef MATTER_CLUSTER_CONSTANTS_H
+#define MATTER_CLUSTER_CONSTANTS_H
+
+#import <Foundation/Foundation.h>
+#import <stdint.h>
+
+#pragma mark - Clusters IDs
+
+typedef NS_ENUM(uint32_t, MatterClusterIDType) {
+{{#zcl_clusters}}
+kMatterCluster{{asUpperCamelCase label}}ID = {{asMEI manufacturerCode code}},
+{{/zcl_clusters}}
+};
+
+#pragma mark - Attributes IDs
+
+typedef NS_ENUM(uint32_t, MatterClusterAttributeIDType) {
+// Global attributes
+{{#zcl_attributes_server}}
+{{#unless clusterRef}}
+kMatterClusterGlobalAttribute{{asUpperCamelCase label}}ID = {{asMEI manufacturerCode code}},
```

Comment:
For command payloads we currently use "CHIP{{asUpperCamelCase clusterName}}Cluster{{asUpperCamelCase commandName}}Params" as the type, and structs are "CHIP{{asUpperCamelCase clusterName}}ClusterStructName".

And I guess for enums/bitmaps we use "CHIP{{asUpperCamelCase clusterName}}{{asUpperCamelCase typeName}}" without "Cluster" anywhere in there.... :(

We should probably do a followup to see if we can rationalize these various naming conventions.

Reply:
Sounds good.